ui/src/app/features/model-list/model-list.component.ts

```diff
@@ -73,7 +73,7 @@ export class ModelListComponent implements OnInit, OnDestroy {
     const dialog = this.dialog.open(EditDialogComponent, {
       width: '400px',
       data: {
-        entityType: 'model',
+        entityType: 'Face Collection',
```

Comment:
This should come from translations. Need to refactor all modals titles, but it is out of the scope of this ticket. I'll create a new one for that.